sql-parser/src/main/java/io/crate/sql/tree/ShowSessionParameter.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.sql.tree;
+
+import javax.annotation.Nullable;
+import java.util.Objects;
+
+public class ShowSessionParameter extends Statement {
+
+    @Nullable
+    private final QualifiedName parameter;
+    private final boolean all;
+
+    public ShowSessionParameter(@Nullable QualifiedName parameter, boolean all) {
```

Comment:
can't we deduct the `all` value from the `parameter` value (i.e. `parameter == null`)?